yaml/places.yaml

```diff
@@ -39,6 +39,42 @@ global:
         - [ 9, 9 ]
       default: 10
 
+  - &ne_country_capital [Admin-0 region capital, Admin-0 capital alt, Admin-0 capital]
+  - &ne_region_capital [Admin-1 capital, Admin-1 region capital]
+
+  - &alternate_fclass
+    fclass_iso: { col: fclass_iso }
+    fclass_us: { col: fclass_us }
```

Comment:
after ISO these should probably sort a-Z? 

(I regret not doing that in the SHP's DBF)

Reply:
Yup, makes them easier to read / find stuff. Fixed in c6dae6d7.